### ISSUE TITLE
Update Tooltip bg Color

### ISSUE THREAD

**Issue by adriennetilley** [HausDAO/daohaus-monorepo#543]
### Feature

component library

### Expected outcome

Adjust the tooltip background to: 
rgb(34, 52, 110);
#22346E
Indigo/dark/6

### Actual outcome

The tooltips meld in with the input background too much, it's hard to distinguish them visually. 

### Browser

Chrome

### Screenshots and context

<img width="575" alt="Screen Shot 2022-07-29 at 8 44 01 AM" src="https://user-images.githubusercontent.com/42706432/181763202-f61c086a-0957-4cfe-991f-8aaf8b6840f6.png">

**Comment by earth2travis** [comment]
<img width="618" alt="image" src="https://user-images.githubusercontent.com/522182/185640767-5662fa69-8c3a-45b4-bd57-8e61859c69e7.png">

The input and the tooltip are both using the same color value in theme.ts

```
 dropdown: DropdownDark,
  tooltip: {
    bg: secondaryDark.tooltipBg,
  },
```
Added `tooltipBg: indigoDark.indigo6` to `secondaryDark`

If this is the approach we want to take we will need to add `tooltipBg` to all the other color objects in `colors.ts`

Noticed that the dropdown.ts model has a different way of handling this. Do we need to add a tooltip.ts molecule?

@jonathanprozzi @jordanlesich @0xRowdy

**Comment by earth2travis** [comment]
@0xRowdy do you have any feedback on this? Or is this an issue you would be interested in completing?

**Comment by 0xRowdy** [comment]
> @0xRowdy do you have any feedback on this? Or is this an issue you would be interested in completing?

I believe that we pointed that out to design early on. We can use the varible name 'elementBorderSubtle' as it is equal to Indigo/dark/6. I've got an action item in the backlog to rename all of our color variables to step1 - step12. I've gone ahead and fixed this in this [PR](https://github.com/HausDAO/daohaus-monorepo/pull/693) along with another tooltip related bug.
